internal/app/app.go

```diff
@@ -37,7 +37,7 @@ type githubClient interface {
 	GetLatestCommitFromBranch(ctx context.Context, owner, repo, branch string) (string, error)
```

Comment:
Good addition to include both `startSide` and `side` in the method signature for `AddCommentToPullRequestReview`. This change ensures that the comments can be correctly placed in the diff context of the PR, whether they relate to the left side (base) or the right side (compare).